pbxproj/pbxsections/PBXVariantGroup.py

```diff
@@ -0,0 +1,13 @@
+from pbxproj.pbxsections import PBXFileReference, PBXGroup
+
+
+class PBXVariantGroup(PBXGroup):
```

Comment:
I have added this class and everything works just magically. I have no idea how the parser works. Super magic @kronenthaler!

Where's the actual string parser code though?

Reply:
There are 2 parts on the parsing, one is the parsing of the openstep format to JSON like (that is [here](https://github.com/kronenthaler/openstep-parser)), and the second is mapping the JSON-like to objects, that's pretty much done [here](https://github.com/kronenthaler/mod-pbxproj/blob/master/pbxproj/PBXGenericObject.py#L66). Basically, creating instances of the object with the same class name.

Reply:
Neat!

```
    @classmethod
    def _get_class_reference(cls, class_type):
        module = __import__('pbxproj')
        return getattr(module, class_type, PBXGenericObject)
```